iabtcf-decoder/src/main/java/com/iabtcf/utils/IntIterableUtils.java

```diff
@@ -51,4 +52,13 @@ public static IntStream toStream(final IntIterable it) {
                 it.intIterator(),
                 Spliterator.ORDERED | Spliterator.IMMUTABLE | Spliterator.NONNULL), false);
     }
+
+    public static IntIterable toIntIterable(int... values) {
```

Comment:
BitSetIntIterable#of(int... values) was added in the infrastructure commit c89a00bd1688cc50f7aa0829e6a654367f3f0436